README.md

```diff
@@ -143,6 +143,7 @@ npm i template-typescript-node-package
 ```
 
 ```ts
+// eslint-disable-next-line n/no-missing-import
```

Comment:
Do you think this is okay to comment with eslint disable lin in the README.md, because eslint warns here aswell

Reply:
Nice thought !!

I will add `template-typescript-node-package` in `allowModules`.